src/collections/_documentation/development/sdk-dev/event-payloads/index.md

```diff
@@ -3,6 +3,36 @@ title: Event Payloads
 sidebar_order: 3
 ---
 
+Events are the fundamental data that clients, often through use of an SDK, send
+to the Sentry server.
+
+The API endpoint on the Sentry server where event payloads are sent is
+`/api/{PROJECT_ID}/store/`.
+
+{% capture __alert_content -%}
+We strive to document the canonical format of events and its additional
+interfaces. However, note that for backwards compatibility the server also
+understands events that are not in the canonical format described throughout the
+documentation.
+
+Existing SDKs may be using a historical format that is not recommended for new
+code.
```

Comment:
I want this to be able to explain to people/myself when we see SDK code that doesn't match the documentation.